src/localStorage.js

```diff
@@ -0,0 +1,14 @@
+export const localstorageStoreWelcomeBox = (welcomeBox) => {
+  localStorage.setItem('welcomeBox', welcomeBox);
```

Comment:
Hm, blir kanskje ikke like greit for WelcomeBox-komponenten...
hva med at `lastLesson` også legges i state? Og at denne blir satt i `store.js`, samt i en reducer hver gang man går inn på en lesson? Og så kan WelcomeBox bare lese `lastLesson` ut av state i stedet? Så trenger vi ikke bekymre oss for om localStorage finnes i WelcomeBox.